file_paths.md

```diff
@@ -59,6 +59,11 @@ The files annotated by an '*' indicate that they are created by package installa
         routes.yaml                       # :route_file
         ssl                               # :ssldir                      /etc/puppet/ssl
 
+    /etc/puppetlabs/pxp-agent *
+        modules.d *                       # directory where pxp-agent modules are stored
+            puppet.cfg *                  # /opt/puppetlabs/pxp-agent/modules/puppet configuration
+        pxp-agent.cfg                     # pxp-agent configuration file
```

Comment:
Currently the only configuration value is the path to puppet. If this was to remain the case then I guess we could hardcode it.

I do however feel that leaving the config file around might be the better plan in the long term. Right now the module fails very quietly and I can see value in adding a debug or verbose config file option that will allow us to fail louder, possibly making the module easier to debug for support in the wild.